Purchases/Public/RCPurchases.m

```diff
@@ -1039,7 +1039,26 @@ - (void)storeKitWrapper:(RCStoreKitWrapper *)storeKitWrapper
             }
             break;
         }
-        case SKPaymentTransactionStateDeferred:
+        case SKPaymentTransactionStateDeferred: {
+            RCPurchaseCompletedBlock completion = nil;
+            @synchronized (self) {
+                completion = self.purchaseCompleteCallbacks[transaction.payment.productIdentifier];
+            }
+            NSError *pendingError = [NSError errorWithDomain:RCPurchasesErrorDomain
+                                                        code:RCPaymentPendingError
+                                                    userInfo:@{
+                                                        NSLocalizedDescriptionKey: @"The payment is deferred."
+                                                    }];
```

Comment:
maybe this should live in https://github.com/RevenueCat/purchases-ios/blob/develop/Purchases/Public/RCPurchasesErrorUtils.h